core/primitives/src/epoch_manager.rs

```diff
@@ -280,6 +280,11 @@ impl AllEpochConfig {
 /// algorithm.  See <https://github.com/near/NEPs/pull/167> for details.
 #[derive(Debug, Clone, SmartDefault, PartialEq, Eq)]
 pub struct ValidatorSelectionConfig {
+    #[default(100)]
+    pub num_chunk_producer_seats: NumSeats,
+    #[default(300)]
+    pub num_chunk_validator_seats: NumSeats,
+    // Deprecated, should be set to zero for future protocol versions.
```

Comment:
Can we make this a TODO and also indicate after which protocol versions this can change (in case someone considers doing it later)?